addon/index.js

```diff
@@ -45,13 +46,37 @@ const button = ActionButton({ // eslint-disable-line new-cap
     '32': './icon-32.png',
     '64': './icon-64.png'
   },
-  badge: state.updates.length,
-  badgeColor: prefs.BADGE_COLOR,
   onClick: function() {
     tabs.open({ url: prefs.CONTENT_URL });
   }
 });
 
+function getInstalledAddons(cb) {
+  request({
+    url: 'http://ideatown.dev:8000/api/experiments?format=json',
+    onComplete: res => {
+      if (!res.json.results.length || res.error) {
+        console.error('No experiments:: ', res);
+      }
+
+      store.availableExperiments = res.json.results.map(exp => {
+        return exp.addon_id;
+      });
+
+      AddonManager.getAllAddons(addons => {
+        cb(store.experiments = addons.filter(a => {
+          return isIdeatownAddon(a.id);
+        }).map(a => {
+          return {
+            id: a.id,
+            version: a.version
+          };
+        }));
```

Comment:
`map` and `filter` can make for some pretty cryptic code. Might I recommend something like

```
AddonManager.getAllAddons(addons => {
  const ideas = [];
  addons.forEach(a => {
    if (isIdeatownAddon(a)) {  // note, make isIdeatownAddon smart enough to check the id
      ideas.push({id: a.id, version: a.version});
    }
  });
  cb(ideas);
});
```

Purely stylistic thing, though; keep it this way if you feel strongly about it ^_^